packages/core/src/RenderingEngine/RenderingEngine.ts

```diff
@@ -611,13 +611,13 @@ class RenderingEngine implements IRenderingEngine {
       const resetToCenter = true;
       const resetRotation = false;
       const suppressEvents = true;
-      vp.resetCamera(
+      vp.resetCamera({
```

Comment:
Since it is now self documenting, you can avoid the const declarations and resetCamera({
  resetToCenter: true,
  resetRotation: false,
...})
instead of declaring consts for a single use.

Reply:
Actually, declaring that statically as a value might be worth doing too, so you don't create the object all the time, and more importantly, so you can name the reset action you are calling.